src/Matrix.jl

```diff
@@ -29,6 +29,7 @@ function check_parent(a::MatrixElem, b::MatrixElem, throw::Bool = true)
 end
 
 function _check_dim(r::Int, c::Int, arr::AbstractMatrix{T}, transpose::Bool = false) where {T}
+  Base.require_one_based_indexing(arr)
```

Comment:
Those are not official API. OTOH that hasn't stopped us is in other cases...

Reply:
They are in julia nightly. See https://github.com/JuliaLang/julia/pull/56196